CONTRIBUTING.md

```diff
@@ -105,6 +105,12 @@ npm run dev:start
 
 Enjoy the test drive!
 
+## Dev Setup
+
+When running Jam in dev mode, Additional Jam instances are provided by default. These instances can be used to set up different wallets and simulate collaborative transactions between themselves as though between different identities. A block explorer and RPC terminal are also available for directly interacting with the regtest node that Jam uses.
+
+Information on how to access these development features can be found on the _/dev-setup_ route at port 3000
```

Comment:
```suggestion
Information on how to access these development features can be found at [http://localhost:3000/dev-setup](http://localhost:3000/dev-setup).
```

What do you think about linking to the page directly?

Reply:
I think it's fine to do that. Initially had that idea but ended up not doing it that way. I'll commit that suggestion